### ISSUE TITLE
Adding contao theme installation to the install tool

### ISSUE THREAD

**Issue by may17** [contao/core#7629]
As defined in our last mumble call, about the contao demo, the install tool should be extend with a function to install the contao demo (like the composer client).

**Comment by leofeyer** [comment]
As discussed on Mumble, @samuelll offered to create a separate Contao distribution with the COD included.

**Comment by discordier** [comment]
@samuelll is now @sascha-mueller

**Comment by sascha-mueller** [comment]
Zum Verständnis:
Hieße der oben von Joe beschriebene Weg, dass die Demo dann direkt samt Seitenbaum importiert wird und nutzbar ist?
Leo klang so, als sei dies nicht der Fall.

**Comment by leofeyer** [comment]
Nein, das würde nur heißen, dass die Demo direkt im Installtool angeboten wird und man per Klick direkt ins ER springen könnte. Für den Import des SQL-Dumps müsste man aber wieder zurück ins Installtool.

**Comment by sascha-mueller** [comment]
Ist eine weitere Automatisierung denn technisch so aufwändig?
Nur ins ER zu springen ist vielleicht ein netter erster Schritt, aber bringt die ganz Neuen ebenfalls nicht weiter. Auch hier wird keiner kapieren, dass er für den Import nochmal zurück muss.

**Comment by leofeyer** [comment]
Deswegen hast Du ja angeboten, eine separate Contao-Distribution mit der COD bereitzustellen :)

Das Problem an einem automatischen SQL-Import ist, dass den vermutlich nicht jeder haben will. Also müsste man wieder eine Routine einbauen, die nachfragt, und weil das dann mehrere Requests wären, geht es nicht in einer `runonce.php` und so wird der Ratteschwanz immer länger.

Und dieser Aufwand steht in keinem Verhältnis zu den nur 0,18% der Nutzer, die sich die Demo überhaupt herunterladen (Contao wurde 1.084.755 heruntergeladen, die [COD](https://contao.org/de/erweiterungsliste/view/official_demo.30050009.de.html) 2.026 Mal).

**Comment by aschempp** [comment]
Ich bin auch der Meinung wir sollten einen separaten Demo-Download anbieten. So macht Isotope das auch: https://isotopeecommerce.org/de/demo.html
Ich finde auch die Demo muss nicht zwingend Tagesaktuell sein, um das System zu testen ist es egal ob die Demo 3.5.2 oder 3.5.3 ist…

**Comment by leofeyer** [comment]
@sascha-mueller @may17 @aschempp Wir sollten das mal auf Mumble besprechen.

**Comment by sascha-mueller** [comment]
Dafür. Ich wollte es schon beim letzten Call ansprechen.
Jedoch schien es zu keiner Zeit wirklich rein zu passen.

**Comment by leofeyer** [comment]
Lass uns einfach einen separaten Termin dafür machen.

**Comment by may17** [comment]
Sagt Bescheid wann :) Bei mir geht es meist immer tagsüber.

**Comment by leofeyer** [comment]
Bei mir auch. Vorzugsweise nachmittags.

**Comment by sascha-mueller** [comment]
Wollen wir das dies Jahr noch machen?
Würde dann noch einen Doodle für verschiedene Tage und zb 15Uhr als Termin anlegen.

**Comment by leofeyer** [comment]
Wie sieht es denn hiermit aus? 😄

**Comment by may17** [comment]
Tjo... bin immer noch dafür 😆

**Comment by leofeyer** [comment]
Ich auch. Und ich fände es noch cooler, wenn wir die Demo mit Docker zusammen ausliefern könnten, so dass man sie lokal direkt testen kann.

**Comment by sascha-mueller** [comment]
Ich bin auch noch immer dafür.

**Comment by aschempp** [comment]
> die Demo mit Docker zusammen ausliefern könnten, so dass man sie lokal direkt testen kann.

glaubst du ernsthaft jemand der die Demo nutzen will weiss wie Docker funktioniert? Wenn ich sehen will wie Contao funktioniert dann nehme ich demo.contao.org, nicht?

Ich bin nicht wirklich überzeugt ob das ins Install-Tool gehört. Wäre es nicht sinnvoller wir hätten eine "echte" Installationsroutine oder eine Möglichkeit das über Composer zu machen?

**Comment by leofeyer** [comment]
Die Idee im Verlaufe der Diskussion (siehe oben) war, einen separaten Download einer Installation mit Demo anzubieten. Im Installtool macht es keinen Sinn, da dieses ja weg soll.

> Ich bin auch der Meinung wir sollten einen separaten Demo-Download anbieten. So macht Isotope das auch: https://isotopeecommerce.org/de/demo.html

**Comment by Aybee** [comment]
Eine Demo direkt über das Installtool, oder das BE installieren zu können finde ich sehr gut. So wäre man noch mehr motiviert auch mal eben neuere nicht LTS Versionen zu installieren und zu prüfen.
